src/bundle/Resources/mappings/config-resolver-namespaces-map.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+return [
+    'ezsettings' => 'ibexa.site_access.config',
+    'ezrecommendation' => 'ibexa.personalization.site_access.config',
```

Comment:
Personally I am against separate namespace just for personalization, unless this is temporary because of the overlay of two PRs.
@ciastektk @alongosz 

if anything it should be `ibexa.site_access.config.personalization`

Reply:
> Personally I am against separate namespace just for personalization, unless this is temporary because of the overlay of two PRs.
> if anything it should be ibexa.site_access.config.personalization

@ViniTou I think @ciastektk is gonna merge it into one. On the side note - it cannot be `ibexa.site_access.config.personalization` because it's reserved for: `ibexa.site_access.config.<scope>` and might cause a conflict when someone decides to create `personalization` Site Access.

Reply:
Yes, I will merge both configurations into one. Most of the parameters will follow the pattern: `ibexa.site_access.config.<scope>.personalization`